packages/model-viewer/src/features/annotation/hotspot.ts

```diff
@@ -142,6 +145,10 @@ export class Hotspot extends CSS2DObject {
     }
   }
 
+  orient(radians: number) {
```

Comment:
Is there a test case covering the behavior for this?

Reply:
No, it seemed like putting one on hotspot would be pretty tautological. I'd like to have one testing AR, but I'm not sure we have it sufficiently mocked out yet. What would you like to see?